src/server/system_services/schemas/nsfs_account_schema.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     required: [
         '_id',
         'name',
-        'email',
+        'email', // temp, keep the email internally
```

Comment:
why do we still need to keep it internally?

Reply:
I had issues with tests `test_bucketspace_fs.js`, `test_bucketspace.js`, and `NSFS Ceph S3 tests` when I removed the email from the account totally, so I decided to remove the email in steps. Since the schedule is tight I published the PR with the first step which is removing the email flag from the end user, and keeping it internally. I will try it again.

Reply:
alright, what was the issue you saw?

Reply:
I fixed it, you can see that the tests are passing in the CI (the fix is in 2 separate commits, hopefully for easier code review).